src/main/webapp/user-page.html

```diff
@@ -31,6 +32,8 @@
     </nav>
     <h1 id="page-title">User Page</h1>
 
+
```

Comment:
Remove these 2 extra lines. Extra blank lines are not as problematic as unnecessarily modifying lines, but it's a good habit to keep your change as tidy as possible.